### DISCUSSION TITLE
teoria relazionale - 12

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/basi-di-dati-1#79] [Esercizi]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/5bf10daf-dc15-431d-aa67-79701ac0a3ce)

**Comment by atyion** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/101365532/dfcb514b-a28f-4a3f-8366-9ecea0442a87)

**Comment by IonutCicio** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/1d4d9ffc-811f-47a7-8eb3-9658013c555a)

**Comment by daromacarlo** [comment]
![Immagine 2024-02-04 180320](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/553029c1-70a7-427a-89ed-82312fb9d8ef)

**Comment by Frisbon** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/84731341/035b2b83-97e3-41a4-8bbd-f8f5add9c8e5)
